tests/test_certgen_wiring.py

```diff
@@ -0,0 +1,136 @@
+import unittest
+from collections import namedtuple
+from acm_pca_cert_generator import certgen
+
+try:
+    import mock
+    from mock import MagicMock
+    from mock import call
+    from mock import create_autospec
+except ImportError:
+    from unittest import mock
```

Comment:
I'm not sure all these imports are used. I can only see calls to `MagicMock()`

Reply:
Ahh yeah thanks. Done.